aiomcache/client.py

```diff
@@ -58,6 +59,70 @@ def _validate_key(self, key):
         return key
 
     @asyncio.coroutine
+    def close(self):
+        """Closes the sockets if its open."""
+        yield from self._pool.clear()
+
+    @acquire
+    def get(self, conn, key, default=None, with_flags=False):
+        """Gets a single value from the server.
+
+        :param key: ``bytes``, is the key for the item being fetched
+        :param default: default value if there is no value.
+        :return: ``bytes``, is the data for this specified key.
+        """
+        values, _, flags = yield from self._multi_get(
+            conn, key, with_cas=False, with_flags=with_flags)
+        if with_flags:
+            return values.get(key, default), flags.get(key)
+        else:
+            return values.get(key, default)
```

Comment:
Returning different data types depending on a flag is a bad design. I prefer to return always the same structure even if it contains an empty list rather than this.

Reply:
this is actually good point, better to have separate method for each type or something